EnglishFilter/sections/specific.txt

```diff
@@ -358,6 +358,7 @@ sunporno.com##.atvtabling
 monsterspost.com##.ch_banner
 gamertweak.com##div[class^="Adsinnov_"]
 nytimes.com###app > div.e1xxpj0j0
+space.com###article-body > div.jwplayer__widthsetter
```

Comment:
Actually, this is not an ad. Have you checked the element without AdGuard?
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/144251594-25145289-c405-4600-945f-a0f22cc7b8bd.png)

</details><br/>